roboflow/core/version.py

```diff
@@ -957,7 +943,7 @@ def data_yaml_callback(content: dict) -> dict:
                 content["train"] = location + content["train"].lstrip(".")
                 content["val"] = location + content["val"].lstrip(".")
                 content["test"] = location + content["test"].lstrip(".")
-            if format in ["yolov5pytorch", "yolov7pytorch", "yolov8", "yolov9"]:
```

Comment:
We should test yolov9 as well. I thiiiiink it was yolov5-based? Could be that it needs special treatment like this.

Reply:
Yeah, let's release this and we can test that ;-)